util/src/telemetry/datadog/mod.rs

```diff
@@ -0,0 +1,44 @@
+//! Types and helpers for configuring DataDog telemetry
+
+use std::env;
+
+use super::TelemetrySetupError;
+
+pub mod formatter;
+
+/// The name of the environment variable used to set the
+/// service name for DataDog unified service tagging
+pub const DD_SERVICE_ENV_VAR: &str = "DD_SERVICE";
+/// The name of the environment variable used to set the
+/// environment for DataDog unified service tagging
+pub const DD_ENV_ENV_VAR: &str = "DD_ENV";
+/// The current relayer version, used to tag telemetry resources
+pub const VERSION: &str = env!("CARGO_PKG_VERSION");
```

Comment:
@joeykraut thoughts on using this to replace the `version` cli / config file option?

Reply:
sure